ddtrace/contrib/integration_registry/registry.yaml

```diff
@@ -0,0 +1,592 @@
+  tested_version_min: 0.1.1
+  tested_version_max: 0.2.0
+
+- integration_name: aiopg
+  is_external_package: true
+  dependency_name:
+  - aiopg
+  tested_version_min: 1.4.0
+  tested_version_max: 1.4.0
+
+- integration_name: aioredis
+  is_external_package: true
+  dependency_name:
+  - aioredis
+  is_tested: false
+
+- integration_name: algoliasearch
+  is_external_package: true
+  dependency_name:
+  - algoliasearch
+  tested_version_min: 2.6.3
+  tested_version_max: 2.6.3
+
+- integration_name: anthropic
+  is_external_package: true
+  dependency_name:
+  - anthropic
+  tested_version_min: 0.26.0
+  tested_version_max: 0.49.0
+
+- integration_name: aredis
+  is_external_package: true
+  dependency_name:
+  - aredis
+  tested_version_min: 1.1.8
+  tested_version_max: 1.1.8
+
+- integration_name: asgi
+  is_external_package: false
+
+- integration_name: asyncio
+  is_external_package: false
+
+- integration_name: asyncpg
+  is_external_package: true
+  dependency_name:
+  - asyncpg
+  tested_version_min: 0.23.0
+  tested_version_max: 0.30.0
+
+- integration_name: avro
+  is_external_package: true
+  dependency_name:
+  - avro
+  tested_version_min: 1.12.0
+  tested_version_max: 1.12.0
+
+- integration_name: aws_lambda
+  is_external_package: true
+  dependency_name:
+  - aws_lambda
+  tested_version_min: 6.105.0
+  tested_version_max: 6.105.0
+
+- integration_name: azure_functions
+  is_external_package: true
+  dependency_name:
+  - azure-functions
+  tested_version_min: 1.21.3
+  tested_version_max: 1.21.3
+
+- integration_name: boto
+  is_external_package: true
+  dependency_name:
+  - boto
+  tested_version_min: 2.49.0
+  tested_version_max: 2.49.0
+
+- integration_name: botocore
+  is_external_package: true
+  dependency_name:
+  - botocore
+  tested_version_min: 1.20.106
+  tested_version_max: 1.37.26
+
+- integration_name: bottle
+  is_external_package: true
+  dependency_name:
+  - bottle
+  tested_version_min: 0.12.25
+  tested_version_max: 0.13.2
+
+- integration_name: cassandra
+  is_external_package: true
+  dependency_name:
+  - cassandra-driver
+  tested_version_min: 3.24.0
+  tested_version_max: 3.29.2
+
+- integration_name: celery
+  is_external_package: true
+  dependency_name:
+  - celery
+  tested_version_min: 5.3.6
+  tested_version_max: 5.4.0
+
+- integration_name: cherrypy
+  is_external_package: true
+  dependency_name:
+  - cherrypy
+  tested_version_min: 17.4.2
+  tested_version_max: 18.10.0
+
+- integration_name: consul
+  is_external_package: true
+  dependency_name:
+  - python-consul
+  tested_version_min: 1.1.0
+  tested_version_max: 1.1.0
+
+- integration_name: coverage
+  is_external_package: true
+  dependency_name:
+  - coverage
+  tested_version_min: 7.4.4
+  tested_version_max: 7.4.4
+
+- integration_name: crewai
+  is_external_package: true
+  dependency_name:
+  - crewai
+  tested_version_min: 0.108.0
+  tested_version_max: 0.108.0
+
+- integration_name: ddtrace_api
+  is_external_package: false
```

Comment:
This isn't accurate. The external package for this one is `ddtrace-api` https://pypi.org/project/ddtrace-api/